src/main/java/it/unisa/Amigo/gruppo/services/GruppoServiceImpl.java

```diff
@@ -76,7 +76,7 @@
      * @return lista di supergruppi
      */
     @Override
-    public List<Supergruppo> visualizzaSupergruppi(int idPersona) {
+    public List<Supergruppo> findAllSupergruppi(int idPersona) {
```

Comment:
```suggestion
    public List<Supergruppo> findAllSupergruppiOfPersona(int idPersona) {
```
Quando cambiate il nome usate il refactoring di intelliJ che lo cambia anche nell''interfaccia.